source/sass/buyers-guide/_colors.scss

```diff
@@ -1,5 +1,6 @@
 // Privacy Not Included (PNI) Palatte
 
+$pni-lilac: #d3d5fc;
```

Comment:
let's add this new colour to `styleguide.html`

Reply:
do we have a dedicated PNI section on the styleguide?